.quickstart/quickstart.yml

```diff
@@ -17,6 +17,10 @@ multi_connector_config:
     database_key: twitter_organic_database
     union_schema_key: twitter_organic_union_schemas
     model_variable: social_media_rollup__twitter_enabled
```

Comment:
this may be out of scope for this PR, but should line 16 actually be `twitter_organic`? @fivetran-joemarkiewicz

Reply:
will keep to current naming based on internal discussion